samples/python/gemm_swiglu_mlp_fusion.py

```
@@ -0,0 +1,278 @@
+
+def swiglu_act(x, Wg, Wu):
+    """Fused h = silu(x@Wg^T) * (x@Wu^T) in ONE cuDNN kernel. x:[M,H] Wg,Wu:[I,H].
+
+    Wg/Wu are the [I,H] nn.Linear weights; they enter the GEMM transposed. We bind them
+    as strided ``.t()`` views ([H,I], column-major), so cuDNN reads them transposed with
+    no copy — materializing ``Wg.t().contiguous()`` would add a [H,I] transpose kernel
+    that costs more than the fused GEMM itself.
+    """
+    h = _handle(x.device)
+    M, H = x.shape
+    I = Wg.shape[0]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `I`

(E741)